src/components/ButtonList.js

```diff
@@ -0,0 +1,11 @@
+function ButtonList({ buttonData }) {
+  return (
+    <div className="button_list">
+      {buttonData.map((button) => (
+        <button>{button.name}</button>
```

Comment:
type attribute도 사용해주시면 좋을 것 같아요~
```suggestion
        <button type='button'>{button.name}</button>
```